### PR TITLE
SameSite for System.Web

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes https://github.com/aspnet/AspNetCore.Docs/issues/16026

[Internal review URL](https://review.docs.microsoft.com/en-us/aspnet/samesite/system-web-samesite?branch=pr-en-us-278)

### CODE REVIEW

File: aspnet/toc.yml [hunk 1]
```diff
@@ -3768,6 +3768,8 @@
   items: 
     - name: Whitepapers
       href: whitepapers/overview.md
+    - name: ASP.NET SameSite
+     href: SameSite/system-web-samesite.md
```
Comment (scottaddie) [inline aspnet/toc.yml]:
```suggestion
      href: SameSite/system-web-samesite.md
```

**Comment by Tratcher** [comment]
@StephenMolloy

File: aspnet/SameSite/system-web-samesite.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+## History and changes
+
+SameSite support was first implemented in .NET 4.7.2 using the [2016 draft standard](https://tools.ietf.org/html/draft-west-first-party-cookies-07#section-4.1).
+
+The November 19, 2019 update for Windows to updated .NET 4.8 from the 2016 standard to the 2019 standard. See the following KB's for more information:
+
+* [KB article 4531182](https://support.microsoft.com/help/4531182/kb4531182)
+* [KB article 4524421](https://support.microsoft.com/help/4524421/kb4524421)
+
+ The [2019 draft of the SameSite specification](https://github.com/aspnet/Announcements/issues/390):
```
Comment (Tratcher) [inline aspnet/SameSite/system-web-samesite.md]: That announcement is specific to Core
```suggestion
 The 2019 draft of the SameSite specification:
```

File: aspnet/SameSite/sample/SameSiteCheck.cs [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+namespace WebApplication64
+{
+    public class SameSiteCheck
+    {
+        #region snippet
+        private void CheckSameSite(HttpContext httpContext, HttpCookie cookie)
+        {
+            if (cookie.SameSite == SameSiteMode.None)
+            {
+                var userAgent = httpContext.Request.Headers["User-Agent"].ToString();
```
Comment (Tratcher) [inline aspnet/SameSite/sample/SameSiteCheck.cs]:
```suggestion
                var userAgent = httpContext.Request.UserAgent;
```

File: aspnet/SameSite/system-web-samesite.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+> * It is not maintained or supported.
+
+[!code-csharp[](sample/SameSiteCheck.cs?name=snippet2)]
+
+## Test apps for SameSite problems
+
+Apps that interact with remote sites such as through third-party login need to:
+
+* Test the interaction on multiple browsers.
+* Apply the [CookiePolicy browser detection and mitigation](#sob) discussed in this document.
```
Comment (Tratcher) [inline aspnet/SameSite/system-web-samesite.md]:
```suggestion
* Apply the [browser detection and mitigation](#sob) discussed in this document.
```

**Pull request merged**